docs/source/routing/about-router.mdx

```diff
@@ -146,4 +160,3 @@ Cloud-hosted routers automatically have access to additional GraphOS Router feat
 
 - To learn more about the intricacies of query plans, see the [example graph](/graphos/reference/federation/query-plans#example-graph) and [query plan](/graphos/reference/federation/query-plans#example-graph) in reference docs
 
-- For the most performant query planning, configure and use the [Rust-native query planner](/graphos/routing/query-planning/native-query-planner).
```

Comment:
this is the actual change i am making in this file. everything else got auto-formatted.